templates/post.html

```diff
@@ -87,6 +99,70 @@ <h3>
     </div>
   </div>
 </div>
+<div class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-8">
+        <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_1212" novalidate="novalidate">
+          <h2>Sign up for email updates</h2>
+          <p>Choose the topics you're interested in</p>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightscloudserver" name="insightscloudserver" value="Cloud and server" {% if post.topics[0].slug == 'cloud' or 'server'  %} checked="checked"{% endif %} />
+                <label for="insightscloudserver">Cloud and server</label>
+              </li>
+              <li class="p-list__item">
+                <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightsdesktop" name="insightsdesktop" value="Desktop"{% if post.topics[0].slug == 'desktop' %} checked="checked"{% endif %} />
+                <label for="insightsdesktop">Desktop</label>
+              </li>
+              <li class="p-list__item">
+                <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightsiot" name="insightsiot" value="Internet of Things"{% if post.topics[0].slug == 'internet-of-things' %} checked="checked"{% endif %} />
+                <label for="insightsiot">Internet of Things</label>
+              </li>
+              <li class="p-list__item">
+                  <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" id="insightstutorials" name="insightstutorials" value="Tutorials" {% if post.topics[0].slug == 'tutorials' %} checked="checked" {% endif %}type="checkbox">
+                  <label for="insightstutorials">Tutorials</label>
+              </li>
+            </ul>
+
+          <div class='mktFormReq mktField'>
+            <label for="Email" class="mktoLabel">Work email: <span>*</span></label>
+            <input required  id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField  mktoRequired" />
+          </div>
+
+          <div class="mktField mktLblRight">
+            <span class="mktInput mktLblRight">
+              <input class="mktFormCheckbox" name="canonicalUpdatesOptIn" id="canonicalUpdatesOptIn" value="Receive updates" tabindex="9" type="checkbox" />
```

Comment:
This tabIndex is making it impossible to tab too

Reply:
Fixed